book/intro.md

```diff
@@ -13,5 +13,5 @@
 :class: seealso
 * JupyterHub: {{ jupyterhub_url }}
 * GitHub organization: {{ github_org_url}}
-* Projects Spreadsheet: {{ project_spreadsheet_url }}
+* [List of projects](projects/list_of_projects)
```

Comment:
I agree with this change - we should discuss if there will be some other resource or link for participants to see a list of proposed projects, since here we will only list those that have turned into active projects. Probably we'll just do that on Slack.